manifests/m/Microsoft/WingetCreate/1.2.6.0/Microsoft.WingetCreate.installer.yaml

```diff
@@ -0,0 +1,43 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=QUSU.CRLF.7-3-4.Win32NT
```

Comment:
Ironic.

Reply:
In my defence I did try creating it with winget-create first but there are still some bugs that need to be squashed there. 

Also unresolved review comments block merge these days..😄